packages/fx-core/tests/plugins/resource/apiconnector/scaffold.test.ts

```diff
@@ -46,6 +46,29 @@ describe("Api Connector scaffold sample code", async () => {
     await fs.remove(testpath);
     sandbox.restore();
   });
+  it("scaffold api without api active resource", async () => {
+    const expectInputs = {
+      component: ["api"],
+      alias: "test",
+      endpoint: "test.endpoint",
+      "auth-type": "basic",
+      "user-name": "test account",
+    };
+    const context = MockContext();
+    context.projectSetting.solutionSettings.activeResourcePlugins = ["fx-resource-bot"];
+    const fakeInputs: Inputs = { ...inputs, ...expectInputs };
+    const apiConnector: ApiConnectorImpl = new ApiConnectorImpl();
+    try {
+      await apiConnector.scaffold(context, fakeInputs);
+    } catch (err) {
+      expect(err instanceof UserError).to.be.true;
+      chai.assert.strictEqual(err.source, "api-connector");
+      chai.assert.strictEqual(
+        err.displayMessage,
+        "Component fx-resource-function not exist, please add first"
```

Comment:
can we should "api" instead of "fx-resource-function" in display message? It would be easier for users to understand the message.